### ISSUE TITLE
Latest Node.js interpreter on Windows cannot import accessibility-insights-report package

### ISSUE THREAD

**Issue by jamalmazrui** [microsoft/accessibility-insights-web#3426]
I am trying to create a Node.js-based command-line utility that takes an Axe results.json file as a parameter and produces an HTML report in the format of the Microsoft Accessibility Insights for Web browser extension. I think the following Node package should make this possible:

https://www.npmjs.com/package/accessibility-insights-report 

However, I cannot successfully import the accessibility-insights-report
package without errors in a test.js script passed to the node interpreter. Here are three attempts.

Attempt 1 (the syntax used in the package's ReadMe file): 

import { reporterFactory } from "accessibility-insights-report";

Attempt 2 (syntax suggested by the node interpreter as an alternative): 

import pkg from 'accessibility-insights-report'; 
const { reporterFactory } = pkg;

attempt 3 (common syntax for loading a Node module): 

const reporterFactory = require('accessibility-insights-report'); 

My web research suggests that the problems encountered are related to differences between S6 and CommonJS syntax for importing a module. I have tried a simple, suggested tweak of renaming my test.js file
to test.mjs (for experimental S6 syntax), but nothing has worked so far.

## Describe the bug

**To Reproduce**

2. Click on '...'
3. Scroll down to '...'

## Context (please complete the following information)

- OS Name & Version: 
- AI-Web Version & Environment: 
- Browser Version: 
- Target Page:

**Comment by ferBonnin** [comment]
Moving this to feature request to have a conversation about this

**Comment by juchampa** [comment]
Thanks @jamalmazrui for your feature request! Can you please respond to the following questions to help us better understand your request: 
1. Is your feature request related to a problem? Please describe it.
2. Describe the desired outcome you expect.
3. Describe any alternatives you’ve considered so far.
4. Any additional context you can provide about this feature request.

**Comment by jamalmazrui** [comment]
I already answered most of these questions, and the initial reply 
implied an understanding of the issue, but let me try again.


1. Is your feature request related to a problem? Please describe it.


This is about a Node.js package that I referenced by URL.  Its purpose 
is to create an HTML report in the same format as the Microsoft 
Accessibility Insights browser extension.  The ReadMe file includes 
sample JavaScript code that presumably should be valid in a script file 
that is run with the node command-line interpreter.  I found, however, 
that the sample line of code for importing a key module of the package 
produces an error about invalid syntax.


2. Describe the desired outcome you expect.


I expect that the sample code should import the module successfully, and 
that the result of a test script, e.g., from running


node test.js


should be an HTML report in the format mentioned.


3. Describe any alternatives you’ve considered so far.


My description included three different syntax alternatives that I had 
attempted to fix the bug, without success.


4. Any additional context you can provide about this feature request.


My description mentioned that web research suggests that the problem is 
related to different forms of syntax that may be used when importing 
JavaScript modules.  I think the one used in the ReadMe file might be 
correct for an import into a web page but not for an import into a Node 
script.  I am not sure about this, however.

**Comment by jamalmazrui** [comment]
Apologies for my delay in responding.  I created the following text 
files as directed:  package.json, tsconfig.json, webpack.config.js, and 
index.ts.  I then ran the CLI commands (on Windows 10) as directed, 
without success.  Here are results from the yarn commands (the node 
command did not work either, but I figure the yarn one should be solved 
first):

yarn install
This seemed to work without errors.

yarn build
This produced output with errors, including one located at the last 
right brace character on the following line:

     const results: AxeResults = {...};

Here is the complete output from the build command:

yarn run v1.22.10
warning package.json: No license field
$ webpack --config webpack.config.js
[webpack-cli] [32mCompilation finished[39m
asset [1m[32mmain.js[39m[22m 2.83 KiB [1m[33m[compared for 
emit][39m[22m (name: main)
[1m./index.ts[39m[22m 647 bytes [1m[33m[built][39m[22m 
[1m[33m[code generated][39m[22m [1m[31m[1 error][39m[22m

[1m[31mERROR[39m[22m in [1m./index.ts[39m[22m [1m[32m7:26[39m[22m
[1mModule parse failed: Unexpected token (7:26)
File was processed with these loaders:
  * ./node_modules/ts-loader/index.js
You may need an additional loader to handle the result of these loaders.
| async function run() {
|     const reporter = accessibility_insights_report_1.reporterFactory();
 >     const results = { ... };
|     const html = reporter.fromAxeResult({
|         results,[39m[22m

[1m[31mERROR[39m[22m in [1mC:\temp\ai_test\index.ts[39m[22m
[1m./index.ts[39m[22m
[1m[tsl] ERROR in C:\temp\ai_test\index.ts(7,37)
       TS1109: Expression expected.[39m[22m

[1mtest[39m[22m (webpack 5.9.0) compiled with [1m[31m2 
errors[39m[22m in 3401 ms
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about 
this command.
